components/dot_menu/index.ts

```diff
@@ -57,9 +57,9 @@ type Props = {
     handleCommentClick?: React.EventHandler<React.MouseEvent | React.KeyboardEvent>;
     handleCardClick?: (post: Post) => void;
     handleDropdownOpened: (open: boolean) => void;
-    handleAddReactionClick: () => void;
+    handleAddReactionClick?: () => void;
```

Comment:
Same here, the search_results_item component uses DotMenu without defining the callback.